conf/configuration.go

```diff
@@ -287,6 +288,10 @@ func (config *Configuration) ApplyDefaults() {
 		config.Cookie.Duration = 86400
 	}
 
+	if config.SwaggerFilePath == "" {
+		config.SwaggerFilePath = "/tmp/swagger.json"
```

Comment:
It'd be nice if this path matched the swagger output path. The standardized golang project structure is to put generated docs in the `docs` directory. I think changing both commands to `docs/swagger.json` makes sense and would follow this standard.